### PR TITLE
module migrated from utils to SDK funcitons

### PR DESCRIPTION
**Pull request by mogith-pn**
https://clarifai.atlassian.net/browse/TT-2956

### CODE REVIEW

File: pages/investigate.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from clarifai.client import create_stub
```
Comment (sainivedh) [inline pages/investigate.py:7]: Please make sure to change the imports to new SDK
```python
from clarifai.client.auth import create_stub
from clarifai.client.auth.helper import ClarifaiAuthHelper
```

File: utils/upload_utils.py [hunk 2]
```diff
@@ -40,9 +41,13 @@ def post_texts_with_geo(
         )
 
         post_inputs_response = stub.PostInputs(post_inputs_request)
```
Comment (sainivedh) [inline utils/upload_utils.py:43]: Any reason on why`PostInputs` is still present. You are uploading inputs using SDK below
```python
input_obj = Inputs(user_id=userDataObject.user_id, app_id = userDataObject.app_id)
post_inp=input_obj.upload_inputs(inputs)
```

Comment (sainivedh) [reply utils/upload_utils.py:43]: I think we can refactor forming input_proto from [get_text_input](https://github.com/Clarifai/clarifai-python/blob/2a662e77fa8d3f6880f28c8792b0bee4fb628cf1/clarifai/client/input.py#L259)
```python
inp = resources_pb2.Input(
                data=resources_pb2.Data(
                    text=resources_pb2.Text(raw=text),
                    geo=resources_pb2.Geo(
                        geo_point=resources_pb2.GeoPoint(
                            longitude=geo_points_batch[idx]["lon"],
                            latitude=geo_points_batch[idx]["lat"],
                        )
                    ),
                )
            )
```

File: pages/investigate.py (hunk 1 above)
Comment (mogith-pn) [reply pages/investigate.py:7]: Modified.

File: utils/upload_utils.py [hunk 3]
```diff
@@ -20,31 +22,27 @@ def post_texts_with_geo(
...
+                geo_info=[geo_points_batch[idx]["lon"],
+                          geo_points_batch[idx]["lat"]
+                          ]
+             )
+            
+            ParseDict(metadata_batch[idx], input_protos.data.metadata)
+            inputs.append(input_protos)
+        
+        #SDK (but returns only jobID)
+        input_obj = Inputs(logger_level="ERROR",
```
Comment (sainivedh) [inline utils/upload_utils.py:39]: Can reuse the same `inp_obj` by intialising above with user_id and app_id

File: utils/upload_utils.py [hunk 4]
```diff
@@ -20,31 +22,27 @@ def post_texts_with_geo(
...
-        post_inputs_request = service_pb2.PostInputsRequest(
-            user_app_id=userDataObject, inputs=inputs
-        )
-
-        post_inputs_response = stub.PostInputs(post_inputs_request)
-        # print(response)
-        if post_inputs_response.status.code != status_code_pb2.SUCCESS:
-            raise Exception("PostInputs request failed: %r" % post_inputs_response)
+            input_obj = Inputs(logger_level="ERROR")
+            id = hashlib.md5(text.encode("utf-8")).hexdigest()
```
Comment (sainivedh) [inline utils/upload_utils.py:26]:
```suggestion
            id = uuid.uuid4().hex
```
this should be random, not depend on text as for same text and diff geo locations. Id remains same and `PostInputs` throws an error.

File: utils/upload_utils.py [hunk 5]
```diff
@@ -58,25 +56,29 @@ def post_texts(st, stub, userDataObject, text_list, metadata_list):
 
         inputs = []
         for idx, text in enumerate(text_batch):
+            input_obj = Inputs(logger_level="ERROR")
+            id = hashlib.md5(text.encode("utf-8")).hexdigest()
```
Comment (sainivedh) [inline utils/upload_utils.py:60]:
```suggestion
            id =  uuid.uuid4().hex
```

File: utils/upload_utils.py [hunk 6]
```diff
@@ -58,25 +56,29 @@ def post_texts(st, stub, userDataObject, text_list, metadata_list):
 
         inputs = []
         for idx, text in enumerate(text_batch):
+            input_obj = Inputs(logger_level="ERROR")
```
Comment (sainivedh) [inline utils/upload_utils.py:59]: Could be initialised at the beginning of func with user_id and app_id, not required for every loop

File: utils/upload_utils.py [hunk 7]
```diff
@@ -58,25 +56,29 @@ def post_texts(st, stub, userDataObject, text_list, metadata_list):
...
-        post_inputs_response = stub.PostInputs(post_inputs_request)
-        if post_inputs_response.status.code != status_code_pb2.SUCCESS:
-            raise Exception("PostInputs request failed: %r" % post_inputs_response)
-
-    return post_inputs_response
+            ParseDict(metadata_batch[idx], input_protos.data.metadata)
+            inputs.append(input_protos)
 
+        #SDK (but returns only jobID, although function is not widely used)
+        input_obj = Inputs(logger_level="ERROR",
```
Comment (sainivedh) [inline utils/upload_utils.py:76]: Can reuse the `input_obj` not required to re-initialise

**Review by sainivedh — COMMENTED** [review]
Added comments on `id` generation and `input_obj` initialisation

File: utils/upload_utils.py [hunk 8]
```diff
@@ -20,31 +22,27 @@ def post_texts_with_geo(
...
-
-        post_inputs_request = service_pb2.PostInputsRequest(
-            user_app_id=userDataObject, inputs=inputs
-        )
-
-        post_inputs_response = stub.PostInputs(post_inputs_request)
-        # print(response)
-        if post_inputs_response.status.code != status_code_pb2.SUCCESS:
-            raise Exception("PostInputs request failed: %r" % post_inputs_response)
+            input_obj = Inputs(logger_level="ERROR")
```
[the remark this answers is not in the document]
Comment (mogith-pn) [reply utils/upload_utils.py]: Declared the input_obj outside loop in the function.

File: pages/investigate.py (hunk 1 above)
Comment (sainivedh) [reply pages/investigate.py:7]: This is not yet modified

`from clarifai.client.auth import create_stub`

Comment (sainivedh) [reply pages/investigate.py]: I meant both as commented on first
for any auth related imports it should be from
`clarifai.client.auth` as we will remove `client.auth` in near future

Comment (mogith-pn) [reply pages/investigate.py]: Hi @sainivedh, 
Some problem with my vscode, that's why it was not committed before. Now I have modified it. Kindly check now.

**Review by sainivedh — APPROVED** [review]
Looks Good.

**Pull request merged**